companies/exor.json

```diff
@@ -0,0 +1,19 @@
+{
+    "slug": "bitel",
+    "relevant-countries": [
+        "nl"
+    ],
+    "categories": [
+        "Holding company"
+    ],
+    "name": "Exor N.V.",
+    "address": "Gustav Mahlerplein 25, 1082 MS Amsterdam, The Netherlands",
```

Comment:
Please delimit the lines of the address with newlines instead of commas. You can also drop the "The" from the country.